src/main/java/examples/LoadCsvDataSetWithTreeMap.java

```diff
@@ -0,0 +1,42 @@
+package examples;
+import dataloader.CsvDataLoader;
+import datastructs.NumericSample;
+
+import java.io.IOException;
+import java.util.TreeMap;
+
+
+/**
+ *
+ * Category: Statistics, Data structures, Data loaders
+ * ID: LoadCsvDataSetWithTreeMap
+ * Description: Load a CSV file using the CsvDataLoader.MapLoader and create
+ * a NumericSample from the data set
+ */
+
+public class LoadCsvDataSetWithTreeMap {
```

Comment:
will you create examples for tablesaw or shall I do it?

Reply:
It's up to you :) :) do you think you have time to do it???

Reply:
I 'll try later on I think. now I don't think I have the time.